tests/setup.py

```diff
@@ -51,7 +51,7 @@ def run_tests(self):
       license="apache2.0",
       packages=find_packages(),
       include_package_data=True,
-      install_requires=["ducktape==0.11.4", "requests==2.31.0"],
+      install_requires=["ducktape==0.8.14", "requests==2.24.0"],
```

Comment:
This is a workaround to get system test workflow running. Need to revert it before merging the PR.